hvantk/algorithms/cohort/frame.py

```
@@ -0,0 +1,234 @@
+    """
+    header = read_header(manifest.table)
+    check_key_column_exists(manifest, header)
+    check_declared_columns_exist(manifest, header)
+
+    df = _read_table(manifest)
+    _check_no_null_genes(df[manifest.key_column], manifest)
+    _check_no_duplicate_genes(df[manifest.key_column], manifest)
+
+    cols = manifest.declared_columns() if include_prior else manifest.axis_columns()
+    declared = [c for c in cols if c != manifest.key_column]
+    frame = df[[manifest.key_column] + declared].rename(
```

Ruff (0.15.21)

[warning] Consider `[manifest.key_column, *declared]` instead of concatenation

Replace with `[manifest.key_column, *declared]`

(RUF005)